protractor.conf.js

```diff
@@ -170,12 +181,18 @@ saveReport = async function (filenameReport, alfrescoJsApi) {
     }
 };
 
-if (LIST_SPECS.length==0) {
-    arraySpecs = [specsToRun];
-} else {
-    arraySpecs = LIST_SPECS.split(',');
-    arraySpecs = arraySpecs.map( (el) => './'+el);
-}
+let specs_to_execute_list = function () {
+    let specsToRun = './**/e2e/' + FOLDER + '/**/*.e2e.ts';
+
+    if (LIST_SPECS.length == 0) {
+        arraySpecs = [specsToRun];
+    } else {
+        arraySpecs = LIST_SPECS.split(',');
+        arraySpecs = arraySpecs.map((el) => './' + el);
```

Comment:
It would be nice in a future iteration to map over this array and complete the path if only the file name is defined. Right now in order to make it work the whole path needs to be added.